tests/ballerina-test/src/test/java/org/ballerinalang/test/object/ObjectAccessWithIndexTest.java

```diff
@@ -131,7 +131,7 @@ public void testUndeclaredObjectAccess() {
     @Test(description = "Test accessing an undeclared field of a object")
     public void testUndeclaredFieldAccess() {
         // TODO change error msg
-        BAssertUtil.validateError(negativeResult, 1, "undefined field 'id' in struct 'Department'", 9, 5);
+        BAssertUtil.validateError(negativeResult, 1, "invalid usage of record literal with type 'Department'", 8, 22);
```

Comment:
The original test case was to check accessing a field which is not there. So we can change the assertion.